nomad/structs/structs.go

```diff
@@ -1139,6 +1148,9 @@ type Node struct {
 	// Raft Indexes
 	CreateIndex uint64
 	ModifyIndex uint64
+
```

Comment:
The Raft indexes are kept at the bottom of the structs so put this above them. I would put it above attributes.